en/config/actuators.md

```diff
@@ -313,6 +313,51 @@ Configuring the Camera Capture / Trigger input requires a reboot to take effect
 You should assign functions to the outputs that match your physical wiring of motors and servos, and use the [Actuator Testing](#actuator-testing) section described below to determine appropriate output parameter values.
 These steps are covered in [Output Assignment and Configuration](#output-assignment-and-configuration).
 
+### Output Functions
+
+Output functions are used to map the "logical functions" of an airframe, such as `Motor 1` or `Landing gear`, to physical outputs like FMU output pin 2.
+This makes it easy to use a particular output pin for almost any purpose.
+
+Some functions are only relevant to particular frames or output types, and will not be offered on others.
+
+Functions include: 
+
+- `Disabled`: Output has no assigned function.
+- `Constant_Min`: Output set to constant minimum value.
```

Comment:
My assumption is that this will be set to whatever the minimum value field is set in the output right? While the constant_max would be set to the max field?
When would you want to set constant values?

Reply:
Thanks. I'll note the value set. But not the reason - since it is a bit speculative.